api/proto/auth/constants.go

```diff
@@ -0,0 +1,34 @@
+package auth
```

Comment:
I move these constants as well as some others to `teleport/constants.go` in #5062